2022-08-trading-orders.md

```diff
@@ -0,0 +1,43 @@
+---
+title: The basics of trading - The orders
+subtitle: 
+slug: trading-orders
+tags: crypto, trading, strategy, automation, orders, blog
+cover: https://images.unsplash.com/photo-1642751226315-e6dc6b47fd54?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=3774&q=80
+domain: blog.octobot.online
+--- 
+
+Trading cryptocurrencies can be exciting, but also take a lot of time if you don’t know how to do it. So, there are many different types of orders you can make to buy or sell on your exchange  according to what you want. Read our introductory article and get a basic understanding of the orders available.
+
+# Market order, the easiest
+
+When you place a market order, you're telling your broker to buy or sell shares at the best available price. If you're buying, your broker will try to get the lowest possible price. If you're selling, they'll try to get the highest. 
+
+That's it! You don't have to worry about timing the market perfectly or anything like that. Just remember that if you're buying, you might not get the exact price you want, and if you're selling, you might not get the highest price possible. 
+
+There are other types of orders, but market orders are the easiest to understand and use.
+
+# Limit order, the most valuable  
+
+When it comes to trading, there are different types of orders that can be placed in order to execute a trade. Among these, the limit order is often considered the most valuable due to its ability to help traders get the best possible price for their desired asset.
+A limit order is a type of trade that gives you exact instructions on when and how much to buy or sell an asset at any time in future. Your trade will be executed either once your price is met or if there are significant changes in the value of this asset.
+
+# Stop order, securing your money
+
+When you're ready to start trading, you need to know the different types of orders that you can place. One important type of order is the stop order. A stop order is an order to buy or sell a security at a specified price. This type of order becomes active only when the security's price reaches the specified price. At that point, the order is executed at the next available price.
+
+Stop orders are often used to limit losses or protect profits. They can also be used to entering or exit a trade. For example, if you wanted to buy a crypto at $50 but it is currently trading at $49, you could place a stop order at $50. If the crypto reaches $50, your order will be executed and you will buy the crypto at $50. Or, if you wanted to sell a crypto at $50 but it is currently trading at $51, you could place a stop order at $50. If the crypto reaches $50, your order will be executed and you will sell the crypto at $50.
```

Comment:
```suggestion
Stop orders are often used to limit losses or protect profits. They can also be used to enter or exit a trade. For example, if you wanted to buy a crypto at $50 but it is currently trading at $49, you could place a stop order at $50. If the crypto reaches $50, your order will be executed and you will buy the crypto at $50. Or, if you wanted to sell a crypto if its price reaches $40 (in order to protect your gains)  while this asset is currently trading at $51, you could place a stop order at $40. If the asset reaches $40, your order will be executed and you will sell the crypto at $40. This would prevent you from holding the asset at lower prices.
```